packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -1209,6 +1214,23 @@ export default {
     updateFilters(updatedFilters) {
       this.$store.commit(SET_SAVED_FILTERS, updatedFilters);
     },
+
+    analysisFindingSelection(findingObject) {
+      const finding = findingObject.resolvedFinding && findingObject.resolvedFinding.finding;
+      if (!finding) return;
+
+      if (finding.checkId === 'n-plus-one-query') {
```

Comment:
I'd avoid the special case and just enable it across the board.

Reply:
Done